admin/refresh_spdx/add_licenses.py

```diff
@@ -1,12 +1,11 @@
+#!/usr/bin/env python3
 """Refresh a BUILD file of license_kinds with new ones from licenses.json.
 
 Usage:
   wget https://github.com/spdx/license-list-data/raw/master/json/licenses.json
-  python add_licenses.py >BUILD.more
-  cat BUILD.more ../../licenses/spdx/BUILD
-
-TODO(aiuto): Refine this enough to update BUILD in place.
-TODO(aiuto): Refer to it from /licenses/spdx/BUILD
+  python add_licenses.py
+  git diff 
```

Comment:
what does `git diff` do here? Is this just for human verification?

Reply:
Yes.